src/main/dist/run_build.sh

```diff
@@ -76,6 +90,16 @@ if [[ -z $ARCHIVE ]]; then
 	ARCHIVE=../r_base_${DIST}_${ARCH}.tar.gz
 fi
 
+if [[ -z $CRAN_URL ]]; then
+  echo "CRAN_URL is not set, using default one: https://cran.asia/"
+  CRAN_URL="https://cran.asia/"
```

Comment:
it used to be `https://cran.mtu.edu/` by default, Is it alright to use `https://cran.asia/` ?

Reply:
`https://cran.mtu.edu/` was unavailable for a couple of weeks, so we decided to use `https://cran.asia/`